cmd/webhook-server/validate.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"errors"
+	"github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"strconv"
+)
+
+func validateUpdatedCephCluster(updatedCephCluster v1.CephCluster, found *v1.CephCluster) error {
+	if isDataDirHostPathChanged(updatedCephCluster, found) {
+		return errors.New("Invalid Update : DataDirHostPath change from " + found.Spec.DataDirHostPath + " to " + updatedCephCluster.Spec.DataDirHostPath + " is not allowed")
```

Comment:
use fmt.Sprintf to format the concat instead

Reply:
Fixed